applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -147,34 +175,69 @@ int le_audio_send(uint8_t const *const data, size_t size)
 	return 0;
 }
 
-// TODO: Implement return code
-void le_audio_enable(le_audio_receive_cb recv_cb)
+int le_audio_enable(le_audio_receive_cb recv_cb)
 {
 	int ret;
 
-	// TODO: Create le_audio_initialize()
-	for (size_t i = 0U; i < ARRAY_SIZE(streams); i++) {
-		streams[i].ops = &stream_ops;
-	}
+	initialize();
+
+	if (broadcast_source == NULL) {
+		LOG_INF("Creating broadcast source");
 
-	LOG_INF("Creating broadcast source");
+		ret = bt_audio_broadcast_source_create(streams, ARRAY_SIZE(streams),
+						       &lc3_preset.codec, &lc3_preset.qos,
+						       &broadcast_source);
+		if (ret) {
+			return ret;
+		}
+	} else {
+		LOG_INF("Broadcast source already created");
+	}
 
-	ret = bt_audio_broadcast_source_create(streams, ARRAY_SIZE(streams), &lc3_preset.codec,
-					       &lc3_preset.qos, &broadcast_source);
-	ERR_CHK_MSG(ret, "Unable to create broadcast source");
+	if (streams[0].ep->status.state != BT_AUDIO_EP_STATE_STREAMING) {
+		LOG_INF("Starting broadcast source");
 
-	LOG_INF("Starting broadcast source");
+		ret = bt_audio_broadcast_source_start(broadcast_source);
```

Comment:
We can currently not use bt_audio_broadcast_stop(), because of a bug (we talked about that you could sniff this). This is why the broadcast source must be in the streaming state all the time now.